app_feup/lib/view/Widgets/generic_expansion_card.dart

```diff
@@ -0,0 +1,45 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:expansion_tile_card/expansion_tile_card.dart';
+
+abstract class GenericExpansionCard extends StatefulWidget {
+  @override
+  State<StatefulWidget> createState() {
+    return GenericExpansionCardState();
+  }
+
+  String getTitle();
+  Widget buildCardContent(BuildContext context);
+}
+
+class GenericExpansionCardState extends State<GenericExpansionCard> {
+  final GlobalKey<ExpansionTileCardState> cardA = GlobalKey();
```

Comment:
Why is this named `cardA`?

Reply:
I did it just to use it in the 'key' argument, but I'm removing it because I can pass GlobalKey to the argument. There was no need to create that variable.